internal/endorser/endorser_test.go

```diff
@@ -103,7 +105,8 @@ func TestGenerateEndorsement_SingleProvenance_EndorseProvenanceLess(t *testing.T
 	}
 	provenances := createProvenanceList(t, []string{"../../testdata/slsa_v02_provenance.json"})
 
-	statement, err := GenerateEndorsement(binaryName, binaryDigestSha256, verOpts, createClaimValidity(7), provenances)
+	digests := map[string]string{"sha256": binaryDigestSha256}
```

Comment:
Should this also be `sha2-256` (and below)?

Reply:
Good catch. Thanks. Fixed.